app/1.0/docs/devguide/local-dom.md

```diff
@@ -10,7 +10,7 @@ its _light DOM_ for clarity.
 
 Polymer supports multiple local DOM implementations. On browsers
 that support shadow DOM, shadow DOM may be used to create local DOM. On other
-supported browsers, Polymer provides local DOM via a custom
+not supported browsers, Polymer provides local DOM via a custom
```

Comment:
@arthurevans In short,  I agree with the solution "On other browsers..." because this expression could be understanded clearly both in english and in chinese. 

However,  between "supported by Polymer" and "that support shadow DOM" , considering the context I tend to think it means the second one both in the original intention, and in the later better fix "On other browsers..." , um... at least look at it in chinese. 

If it refres to "supported by Polymer",then the sentence becomes "On other supported browsers *by Polymer* ...". With the subsequent lines, I really think it doesn't make sense.

Really appreciate for your reading : )